src/services/llms/openai.py

```diff
@@ -0,0 +1,58 @@
+from typing import List, Dict
+from openai import OpenAI
+from unidiff import Hunk, PatchedFile
+from ...core.config import Config
+from ...core.models import PRDetails
+from .base import BaseLLMService
+
+class OpenAIService(BaseLLMService):
+    """
+    Implementation of BaseLLMService for OpenAI's models.
+    """
+    
+    def __init__(self):
+        """Initialize the OpenAI client with configuration."""
+        self.client = OpenAI(api_key=Config.OPENAI_API_KEY)
+        self.model = Config.OPENAI_MODEL
+        
+    def create_prompt(self, file: PatchedFile, hunk: Hunk, pr_details: PRDetails) -> str:
+        """Create a prompt formatted for OpenAI's expectations."""
+        return f"""Your task is reviewing pull requests. Instructions:
+        - Provide the response in following JSON format: {{"reviews": [{{"lineNumber": <line_number>, "reviewComment": "<review comment>"}}]}}
+        - Provide comments and suggestions ONLY if there is something to improve, otherwise "reviews" should be an empty array.
+        - Use GitHub Markdown in comments
+        - Focus on bugs, security issues, and performance problems
+        - IMPORTANT: NEVER suggest adding comments to the code
+
+        Review the following code diff in the file "{file.path}" and take the pull request title and description into account.
+        Pull request title: {pr_details.title}
+        Pull request description:
+
+        ---
+        {pr_details.description or 'No description provided'}
+        ---
+
+        Git diff to review:
+
+        ```diff
+        {hunk.content}
+        ```
```

Comment:
The error handling in `get_ai_response` is insufficient.  Printing the exception to the console is not ideal for production environments.  Consider logging the error with a proper logging library (e.g., `logging`) and potentially raising a more specific exception to be handled by a higher level in the application.  This improves error tracking and debugging.